packages/docz-tools/src/gatsby-theme-docz/components/Sidebar/styles.ts

```diff
@@ -1,6 +1,6 @@
 export const sidebar = (width: number, sidebarOffset: number) =>
   ({
-    bg: "greyBlueDark",
+    bg: "blue",
```

Comment:
these typescript style files don't accept our semantic names, but we can use their values